plugin/trino-hive/src/test/java/io/trino/plugin/hive/AbstractTestHive.java

```diff
@@ -2407,8 +2409,19 @@ public void testPartitionSchemaMismatch()
                 readTable(transaction, table, ImmutableList.of(dsColumn), session, TupleDomain.all(), OptionalInt.empty(), Optional.empty());
             }
         })
-                .isInstanceOf(TrinoException.class)
-                .hasMessageMatching(".*The column 't_data' in table '.*\\.trino_test_partition_schema_change' is declared as type 'double', but partition 'ds=2012-12-29' declared column 't_data' as type 'string'.");
+        .is(new Condition<>(throwable -> {
+            if (throwable instanceof TestAbortedException) {
+                // getTableHandle can throw aborted exception
```

Comment:
This is something I don't understand. Where can TestAbortedException come from and why it's OK here?

Reply:
Sure, I'll make this test skip by overriding it for glue. Other than that, hive hdp3 has passed so these changes are ok, but needs to be improved.